docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,95 @@
+
+NOTE: If you've already installed a threat intelligence integration, you don't need to complete the following steps. Indicator data will automatically display on the Indicators page.
+
+. Choose one of the following:
+* From the {security-app} main menu, go to *Intelligence* -> *Add Integrations*.
+* From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter by threat intelligence integrations.
+. Select a threat intelligence integration, then complete the integration's guided installation.
++
+NOTE: For more information about available fields, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.
+
+. Return to the Indicators page in {elastic-sec}. Refresh the page if indicator data isn't displaying.
+
+[discrete]
+[[troubleshoot-indicators-page]]
+=== Troubleshooting
+If indicator data is not appearing in the Indicators table after you installed a threat intelligence integration:
+
+* Verify that the index holding indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index holding indicator documents will differ based on the way you're collecting indicator data:
+** *{agent} integrations* - If you are collecting indicator data from {agent} threat intelligence integrations, indicator documents are stored in the `logs_ti*` index pattern.
+** *{filebeat} integrations* - If you are collecting indicator data from {filebeat} threat intelligence integrations, indicator documents are stored in the `filebeat-*` index pattern.
+* Make sure the indicator data you're ingesting is mapped to {ecs-ref}[Elastic Common Schema (ECS)].
+
+[discrete]
+[[intelligence-page-ui]]
+== Intelligence page UI
+
+After you add indicators to the Indicators page, you can <<examine-indicator-details, examine>>, search, filter, and take action on indicator data. Indicators also appear in the Trend view, which shows the occurrence of indicators over time.
+
+[role="screenshot"]
+image::images/interact-with-indicators-table.gif[width=90%][height=90%][Shows how to interact with the Intelligence page]
+
+[discrete]
+[[examine-indicator-details]]
+=== Examine indicator details
+Learn more about an IoC by clicking *View details*, then opening the Indicator details flyout. The flyout contains these informational tabs:
+
+* *Overview*: Provides a summary of the indicator, including the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
++
+NOTE: Some threat intelligence feeds provide  https://www.cisa.gov/tlp#:~:text=Introduction,shared%20with%20the%20appropriate%20audience[Traffic Light Protocol (TLP) markings]. The `TLP Marking` field will be empty if the data is not available. The `Confidence` field will also be empty if the data is not available.
```

Comment:
Wouldn't any field, anywhere be empty if the data isn't available? Consider omitting the last two sentences, unless there's some other reason why this needs to be mentioned.

Reply:
I think the idea was to mention that these two fields which are highlighted on the overview tab will always be empty for some feeds as they never provide it. But I agree that if we want to mention it we can do it more clear.

Reply:
Ah, thanks again @maxcold! This does seem good to mention, maybe something like this:
```suggestion
NOTE: Some threat intelligence feeds provide  https://www.cisa.gov/tlp#:~:text=Introduction,shared%20with%20the%20appropriate%20audience[Traffic Light Protocol (TLP) markings]. The `TLP Marking` and `Confidence` fields will be empty if the threat intelligence feed doesn't provide that data.
```

Reply:
1+ for combining the last two sentences and switching to active voice.